tests/conftest.py

```diff
@@ -26,6 +28,16 @@ def clear_all_traces():
     test_exporter.clear()
 
 
+@pytest.fixture(autouse=True)
+def clear_audit_log(request):
```

Comment:
A comment would be helpful -- I don't understand why we need to do this for audit events and not everything else.